src/guide/contributing-to-an-open-jdk-project.md

```diff
@@ -58,6 +58,8 @@ You only need to sign the OCA once in order to cover all changes that you might
 
 Once the OCA is signed, please restrain your urge to create a PR just a little while longer. In order to prepare the community for your patch, please socialize your idea on the relevant [mailing lists]. Almost all changes, and in particular any API changes, must go this route and have a broad agreement in place before there is any point in presenting code. To understand the criteria by which your patch is going to be judged, please read [Why is My Change Rejected?] below. In short, hidden constraints and assumptions, stability and quality, maintainability, compatibility, and conformance to specifications must be considered before your PR is ready to be submitted. If you don't understand the constraints for acceptance, you might be surprised when your PR is rejected.
 
+Please note that lack of engagement should not be interpreted as supporting the proposal. Lack of engagement might be better interpreted as people are busy or maybe that the problem isn't compelling or high priority enough to spend time on right now. If you didn't get the desired attention and the required agreement in your mail thread, **do not** proceed to create a PR.
+
 ### 3. Find a Sponsor
 
 Socializing your change on the mailing lists also prevents the surprise that would otherwise make the community choke on their morning coffee when they see a huge patch in a new, unknown PR. As a new developer in the community you'll need to make a few friends that agree with your change. There are many good reasons to make friends, but the one relevant here is that for your first changes you'll need a [Sponsor](https://openjdk.org/bylaws#sponsor) to facilitate the integration of your work. The [Sponsor](https://openjdk.org/bylaws#sponsor) will perform any number of administrative tasks like JBS updates, additional testing, etc. It's usual for a [Sponsor](https://openjdk.org/bylaws#sponsor) to also be a reviewer of a change and thus familiar with it, but it's not a requirement.
```

Comment:
Imho "for your first changes" is misleading here. I contributed lots of commits, but still do need a sponsor even after years. We should find a wording which does not give the impression that contributing many commits over several years will change anything on the fact that a sponsor *still* is mandatory for each PR.

Reply:
If I'm not mistaken you have contributed 16 changes over the last three and a half year. I haven't looked at them in detail, but provided these aren't trivial changes or changes that caused more harm than good, you should be eligible for a Committer role in the JDK project and thereby not be needing a Sponsor.

Reply:
```
Socializing your change on the mailing lists also prevents the surprise that would otherwise make the community choke on their morning coffee when they see a huge patch in a new, unknown PR.
```

This makes me feel a bit uneasy. To "socialize your change" is much easier for people who are part of a community – in effect, those who are already insiders, but it's hard for a newcomer to know where to start. While I agree that this sentence accurately reflects the reality of the OpenJDK project, socializing is not an easy thing for many people.